portal/src/main/webapp/js/src/oncoprint/geneticrules.js

```diff
@@ -49,7 +49,7 @@ var non_mutation_rule_params = {
 	    legend_label: 'Gain',
 	},
 	// Blue rectangle for deep deletion 
-	'homdel': {
```

Comment:
Same here 
https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/shared/components/oncoprint/geneticrules.ts

Reply:
Just deleted this file (oncoprint directory is obsolete)